reference/content-types/tag_list.rst

```diff
@@ -0,0 +1,26 @@
+Tag list
+========
+
+Description
+-----------
+
+Shows a simple text line with an autocomplete feature for the available Tags in
+the system. Tags, which do not already exist, will be created. Tags can be
```

Comment:
No `,`. `Tags which do not already exist will be created.`.

You could also do this as a note

````
.. note::

    Tags which do not already exist will be created.
````